packages/amplication-client/Dockerfile

```diff
@@ -1,34 +1,18 @@
-ARG NODE_VERSION=16.16-alpine
-ARG NGINX_VERSION=1.20-alpine
-
-FROM node:$NODE_VERSION AS build
-
-ENV OPENCOLLECTIVE_HIDE=1
-
-RUN npm config set fund false
-
-WORKDIR /app
-COPY lerna.json /app
-COPY package*.json /app/
-RUN npm ci --production
-
-COPY packages packages
-RUN npm run bootstrap -- --scope @amplication/client --include-dependencies
-RUN npm run build -- --scope @amplication/client --include-dependencies
-
-FROM nginx:$NGINX_VERSION as client
+FROM nginx:1.20-alpine
 
 RUN apk add --update nodejs
 
 WORKDIR /app
 
-COPY --from=build /app/packages/amplication-client/nginx.conf /etc/nginx/conf.d/default.conf
-COPY --from=build /app/packages/amplication-client/build /app
-COPY --from=build /app/packages/amplication-client/src/scripts/inject-variables.js .
-COPY --from=build /app/packages/amplication-client/entrypoint.sh .
+COPY ./packages/amplication-client/nginx.conf /etc/nginx/conf.d/default.conf
+COPY ./dist/apps/@amplication/client .
+COPY ./packages/amplication-client/src/scripts/inject-variables.js .
+COPY ./packages/amplication-client/entrypoint.sh .
```

Comment:
Ideally we should copy any necessary files during the build process instead of ad-hoc operations in the Dockerfile

Reply:
Had a chat with @overbit and explained to him that this could be resolved using a build target in Nx, and the `assets` property. Unfortunately since we use craco here, I think it's not worth the investment. Resolving this.